src/defaults.ts

```diff
@@ -46,6 +47,8 @@ export const DEFAULT_OPTIONS = {
    * The maximum recursion depth when printing objects in the tooltip.
    */
   maxDepth: 2,
+
+  render: formatValue,
```

Comment:
This needs a description. I also wonder whether there is a better name since `render` sounds like it does more than just generating the HTML.

Reply:
What do you think about a different signature? The signature of `formatValue` is pretty complex and it's easy to implement incorrectly (e.g. not properly sanitizing). I don't have a concrete suggestion but wanted to see if you have an idea.

Reply:
How about `renderTooltip`? We should rename the default method as well. Looking at it now, I think `formatValue` isn't a great name since we also have `valueToHTML` to format a primitive value.  

Also, could you add docs to https://github.com/vega/vega-tooltip/tree/master/docs?